### ISSUE TITLE
Cannot configure plugin OAuth token

### ISSUE THREAD

**Issue by gwihlidal** [alexkvak/teamcity-slack#1]
Hi, I'm trying to use your plugin for slack notifications from team city, but I can't seem to get the configure to work successfully. The plugin installed correctly (I used your prebuilt 1.0 release, uploaded the zip to teamcity and restarted). As directed by the instructions, I went to Administration -> Slack, and checked the Enabled button, which presented me with the OAuth token field. I entered my xoxp-******* token, and clicked Submit. Every time, I get the error "Unable to create session by config" returned to me. I've tried numerous times, verified the app install and OAuth token (another plugin can use it to post to my build channel successfully), and restarted TeamCity multiple times. It would be great to get this plugin working, as I prefer the customization options over the alternatives.

Thank you,
Graham

**Comment by alexkvak** [comment]
Hi,
could you provide screenshot with all config settings you try to set?

**Comment by gwihlidal** [comment]
1)

![image](https://user-images.githubusercontent.com/564042/32435964-d7a59ab8-c2e2-11e7-80f8-283b6b5a550b.png)

2)

![image](https://user-images.githubusercontent.com/564042/32435975-dd66b518-c2e2-11e7-9d5b-e6dcdf072ef3.png)

3)

![image](https://user-images.githubusercontent.com/564042/32435984-e270f186-c2e2-11e7-93f3-9aa10711eb50.png)

**Comment by alexkvak** [comment]
You probably use wrong access token. My token's length is 42, but yours is much longer. Please make sure you use correct token.

**Comment by warrenrumak** [comment]
Be sure to use the "Bot User OAuth Access Token".

**Comment by alexkvak** [comment]
@gwihlidal did you fix it?

**Comment by gwihlidal** [comment]
Hi, I’ll try out the suggestions later tonight. I don’t see a Bot User OAuth Token anywhere specifically but will keep looking.

**Comment by alexkvak** [comment]
You can find it on page "OAuth & Permissions" with URL smth like this
https://api.slack.com/apps/****/oauth

**Comment by gwihlidal** [comment]
Hmm, so I created a bot user which gave me the bot auth token mentioned (i.e. it starts with xoxb and is shorter), and it seemed like it would work, but I got a java error now when trying to proceed further with configuration. This is still just pressing Submit on that same settings page, except using a valid auth token now. I've tried with an without an artifacts server specified too, but that didn't matter.

![image](https://user-images.githubusercontent.com/564042/32521181-d4b8e22a-c412-11e7-8eac-c43f4c36baa1.png)
